THIRD-PARTY-NOTICES.txt

```diff
@@ -0,0 +1,382 @@
+.NET Runtime uses third-party libraries or other resources that may be
+distributed under licenses different than the .NET Runtime software.
+
+In the event that we accidentally failed to list a required notice, please
+bring it to our attention. Post an issue or email us:
+
+           dotnet@microsoft.com
+
+The attached notices are provided for information only.
+
+1. ICU License - ICU 1.8.1 to ICU 57.1
```

Comment:
we're on ICU 67 already. do we really want to mention the individual version number here?